cli/docker-machine.go

```diff
@@ -0,0 +1,26 @@
+package run
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func getDockerMachineIp() (string, error) {
+	machine := os.ExpandEnv("$DOCKER_MACHINE_NAME")
+
+	if machine == "" {
+		return "", nil
+	}
+
+	dockerMachinePath, err := exec.LookPath("docker-machine")
```

Comment:
Can we also check `docker-machine.exe`?  "docker toolbox on windows" installs it as `.exe`

Reply:
Good idea.  It worked for me since I tested using the shell provided by docker toolbox on windows.

Do you mind send a PR that implements this idea?

Reply:
According to this patch, exec.LookPath() on windows should attempt to find executables with .exe or .com extensions. 

https://codereview.appspot.com/13410045/

Reply:
Thanks for the info.  Seems like fairly simple but personally I never used Go before so I'll leave it to others.